src/CommunityToolkit/Xamarin.CommunityToolkit/Views/MediaElement/Tizen/MediaElementExtension.tizen.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Xamarin.Forms;
+using Tizen.Multimedia;
+using ERect = ElmSharp.Rect;
+using MRect = Tizen.Multimedia.Rectangle;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	public static class MediaElementExtension
+	{
+		public static PlayerDisplayMode ToNative(this Aspect aspect)
+		{
+			var ret = PlayerDisplayMode.LetterBox;
+			switch (aspect)
+			{
+				case Aspect.AspectFill:
+					ret = PlayerDisplayMode.CroppedFull;
+					break;
+				case Aspect.AspectFit:
+					ret = PlayerDisplayMode.LetterBox;
+					break;
+				case Aspect.Fill:
+					ret = PlayerDisplayMode.FullScreen;
+					break;
+			}
+			return ret;
```

Comment:
how about adding a `default` case inside the switch and move this line to it. If it's possible can you use the new switch syntax?

Reply:
Absolutely. I will use new switch syntax. Thanks